opentelemetry-api/test-requirements.txt

```
@@ -1,6 +1,6 @@
 asgiref==3.7.2
 Deprecated==1.2.14
-importlib-metadata==6.11.0
+importlib-metadata<=8.2.0
```

```suggestion
importlib-metadata==8.2.0
```

This needs to be `==` for being reproducible

---

I made it `<=` because maybe the newest version is not available in older Python version. But also fine to pin it to exactly one verion.

---

If that's the case we use an older version that works everywhere, if that does not exist we use conditionals but still fixed versions please.